mailbox/memory/src/main/java/org/apache/james/mailbox/inmemory/JsoupTextExtractor.java

```diff
@@ -41,8 +43,9 @@ public ParsedContent extractContent(InputStream inputStream, String contentType,
             return new ParsedContent(IOUtils.toString(inputStream), emptyMetadata);
            }
            if (contentType.equals("text/html")) {
-               String text = Jsoup.parse(IOUtils.toString(inputStream)).text();
-               return new ParsedContent(text, emptyMetadata);
+               Document doc = Jsoup.parse(IOUtils.toString(inputStream));
+               doc.select(TITLE_HTML_TAG).remove();
```

Comment:
- First of all to make it's consistence with Tika, Tika does not contain <title> tag in extracted text. And we have only one IT for both Memory and Cassandra. Of course that we can separated 2 ITs. But the other.
- The second reason, I think it makes sense and all the content does not belong to <body> tag should not belong to extracted text.